<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #000;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Quit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;
}
QPushButton::hover{
	background: #f00;
}</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Party">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_Distance">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>91</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Distance">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>88</width>
     <height>49</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_State">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>181</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 12px; border: 2px inset #ff0;</string>
   </property>
   <property name="text">
    <string>State</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>181</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_IP">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="text">
         <string>Server :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_Port">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="text">
         <string>Port :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_Port">
        <property name="styleSheet">
         <string notr="true">background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;</string>
        </property>
        <property name="minimum">
         <number>1111</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>8888</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Connection">
      <property name="styleSheet">
       <string notr="true">QPushButton {
background: #00008B; font-weight: bold; color: #fff; font-size: 16px; border: 2px inset #ff0;
}
QPushButton::hover{
	background: #0f0;
}</string>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quit</sender>
   <signal>clicked()</signal>
   <receiver>Log</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
